tools/benchmark_tool/openvino/__init__.py

```diff
@@ -50,6 +50,24 @@
 from openvino.runtime import save_model
 from openvino.runtime import layout_helpers
 
+from openvino._pyopenvino import RemoteContext
+from openvino._pyopenvino import RemoteTensor
+
+# OpenCL related:
+try:
+    from openvino._pyopenvino import ClContext
+    from openvino._pyopenvino import ClImage2DTensor
+except ImportError:
+    pass
+
+# libva related:
+try:
+    from openvino._pyopenvino import VADisplayWrapper
```

Comment:
why do we need to expose wrappers to users?

Reply:
Users should be aware of the pointer they are providing.

For OV it means reducing the potential errors as only `VADisplayWrapper` shall be accepted into the function. Not everything that translates to `void *`. Plus it gives space for future improvements without compromising API.